types/chrome/test/index.ts

```diff
@@ -458,6 +458,8 @@ async function testTabGroupInterface() {
         tabGroup.id; // $ExpectType number
         tabGroup.title; // $ExpectType string | undefined
         tabGroup.windowId; // $ExpectType number
+
+        tabGroup.color = 'orange';
```

Comment:
Could you just put a test here for each color mentioned in the docs? Just duplicate this line and replace `orange` with a one of the colors in the docs:
- grey
- blue
- red
- yellow
- green
- pink
- purple
- cyan
- orange